eng/Packages.props

```diff
@@ -37,8 +37,10 @@
     <PackageReference Update="Microsoft.Azure.Cosmos.Table" Version="1.0.7" />
     <PackageReference Update="Microsoft.Azure.KeyVault" Version="3.0.4" />
     <PackageReference Update="Microsoft.Azure.Kusto.Ingest" Version="8.1.0" />
+    <PackageReference Update="Microsoft.Azure.Management.EventHub" Version="2.7.2" />
```

Comment:
We are adding a _lot_ of code that's already deprecated (the entire Microsoft.Azure pile of assemblies isn't really supported anymore and not getting updates). :-(

Reply:
Yeah, it's a really unpleasant situation they are in... They don't want people using the "old" stuff, but the "new" stuff isn't ready yet.  We are doing pretty boring things, so it should be... ok?  If we run into bugs in the old stuff, it's harder to get fixes, since it's either dead (or dying)